vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -158,10 +158,9 @@ void TouchEventHandler::OnPointerConcluded(
   // Only if the view has a Tag can we process this
   int64_t tag;
   winrt::FrameworkElement sourceElement(nullptr);
-  if (!TagFromOriginalSource(args, &tag, &sourceElement))
-    return;
+  if (TagFromOriginalSource(args, &tag, &sourceElement))
+    UpdateReactPointer(m_pointers[pointerIndex], args, sourceElement);
```

Comment:
Can you add comment or refactor this part? It's hard to understand it's not a early return if I didn't look at your PR description.
Maybe you can have better way to do it without comment.
For example: scopeGuard
        auto scopeGuard = gsl::finally([this]()
        {
         }

Reply:
updated comment